dao/client/cp_client.go

```diff
@@ -58,6 +61,14 @@ func NewControlClient(addr string) (s *ControlClient, err error) {
 	s = new(ControlClient)
 	s.addr = addr
 	s.rpcClient = client
+
+	s.timeout = 10*time.Second
+	timeoutvalue := os.Getenv("SERVICED_GETSERVICES_TIMEOUT")
```

Comment:
Typically, we add options like this to the command line as well. Was there discussion about specifically leaving this one off of the command line args?